daemon/k8s_watcher.go

```diff
@@ -77,6 +81,8 @@ var (
 
 	ciliumv1VerConstr, _ = go_version.NewConstraint("< 1.7.0")
 	ciliumv2VerConstr, _ = go_version.NewConstraint(">= 1.7.0")
+
+	cnpCM *controller.Manager
```

Comment:
Oh, if you want to name this k8sCM or something generic then I can use it in #2772  for the `sync-to-k8s-ciliumendpoint-gc` controller

Reply:
done